diff --git a/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/BundleHermesCTask.kt b/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/BundleHermesCTask.kt
index 89ba365..74bcae2 100644
--- a/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/BundleHermesCTask.kt
+++ b/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/BundleHermesCTask.kt
@@ -152,8 +152,7 @@ abstract class BundleHermesCTask : DefaultTask() {
           add("--platform")
           add("android")
           add("--dev")
           add(devEnabled.get().toString())
-          add("--reset-cache")
           add("--entry-file")
           add(entryFile.get().asFile.cliPath(rootFile))
           add("--bundle-output")
